src/components/Home.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable max-len */
+import React from 'react';
+
+function Home() {
+  return (
+    <div className="home-content">
+      <h2>Welcome to our page</h2>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi facere porro voluptatum corporis maxime, dicta dolorem eius exercitationem iste fuga autem esse earum vel libero corrupti debitis quibusdam repellendus similique, quidem, temporibus sequi quis sapiente hic molestiae. Sint similique tempora suscipit itaque! Quam ut reprehenderit minus a error sapiente porro.</p>
+      <p>
+        Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quis suscipit necessitatibus culpa rem, temporibus sed cumque eum aliquam asperiores ipsum ipsa voluptatum iure qui id, totam at officia iste assumenda vitae ad hic sunt exercitationem. Dolorum unde obcaecati hic quae ex temporibus atque itaque voluptates eveniet iusto amet ipsa exercitationem cumque reiciendis quod ut fugit repudiandae assumenda similique, minima voluptate asperiores! Fuga minima laudantium odit maiores corporis molestias doloribus tenetur necessitatibus, doloremque saepe magnam cupiditate praesentium rem, placeat unde magni! Inventore itaque autem unde in praesentium placeat incidunt eos iusto, perspiciatis quidem vitae cupiditate cumque aut quibusdam similique soluta nihil.
+      </p>
+    </div>
```

Comment:
- [ ] `required`
- [ ] great job implementing the full site 👏 
- [ ] however there is need for adjusting in accordance to the template.
- [ ] please move the welcome message as well as the paragraph to the just as the wireframe. 🥂 
- [ ] The layout should match the wireframe please see screenshots below and marked areas.

`your wireframe`
![image](https://user-images.githubusercontent.com/66230306/200111379-e7af6a23-0156-4c12-866d-8885491e6a2d.png)

`original wireframe`
![image](https://user-images.githubusercontent.com/66230306/200111480-8fbf128f-0a0b-48c5-a5d7-c81c8fdd301c.png)